.textlintrc.json

```diff
@@ -210,20 +219,37 @@
         "Ziggo",
         "ZoneMinder",
         [" HA ", "Home Assistant"],
+        ["\\(s\\)he", "they"],
+        ["2 ?steps? authentication", "two-step authentication"],
         ["Alarm ?dot ?com", "Alarm.com"],
         ["an URL", "a URL"],
         ["API['’]?s", "APIs"],
+        ["backwards compatible", "backward compatible"],
+        ["client ?side", "client-side"],
+        ["colour", "color"],
         ["DarkSky", "Dark Sky"],
+        ["end ?to ?end", "end-to-end"],
         ["FRITZ!? ?Box", "FRITZ!Box"],
         ["hass", "Home Assistant"],
+        ["he or she", "they"],
+        ["he/she", "they"],
         ["Home-?Ass?s?iss?tt?ant", "Home Assistant"],
         ["ISY ?994", "ISY994"],
         ["Mac ?OS", "macOS"],
         ["OS ?X", "macOS"],
+        ["repo\\b", "repository"],
+        ["config\\b", "configuration"],
         ["Sky Bell", "SkyBell"],
         ["Sleep IQ", "SleepIQ"],
         ["Twill?o", "Twilio"],
+        ["analyse", "analyze"],
+        ["recognise", "recognize"],
+        ["travelled", "traveled"],
+        ["licence", "license"],
+        ["behaviour", "behavior"],
+        ["two ?steps? authentication", "two-step authentication"],
         ["URL['’]?s", "URLs"],
+        ["walk[- ]through", "walkthrough"],
```

Comment:
Is this a good idea?
I don't know how the linting works. Is the author warned about changes? Is it possible to override?

Reply:
Hmmm, noun vs verb. Probably better to remove it indeed, since you can't override it.

Linting works via CI, the job fails, and it shows you why, which file, which line, which character in the sentence and a suggestion on what to change it into.

Reply:
Fixed.